### ISSUE TITLE
Favorite Inventory isn't Saved

### ISSUE THREAD

**Issue by iFocsy** [Pryaxis/TShock#1866]
* TShock version: self build 83eb707
* TShock build number (if known): 83eb707

1. Have SSC enable in your server.
2. Mark an item as Favorite.
3. Log out.
4. When you are back, the Favorite isn't recovered.

I don't know if it is a Issue or a Request.

#### What plugins and what versions of those plugins are you running?
None, only tshock built-in SSC.

**Comment by AxeelAnder** [comment]
It's technically impossible to implement because of limitation of vanilla terraria.

**Comment by hakusaro** [comment]
We should request this from Re-Logic.

**Comment by dark22** [comment]
> We should request this from Re-Logic.

Yes please. This is a very interesting function of the inventory. It would be great to have it with the SSC.

I don't know if they changed, but the buff time being sent to the server would also be great.

**Comment by Rustly** [comment]
expanding on this idea would be to include modifications to item tooltips/names while you're at asking relogic

**Comment by dark22** [comment]
I don't want to be boring, but the last update of the game, was there any change about that?  If not, did they give any feedback on?

**Comment by hakusaro** [comment]
@dark22 no, the earliest chance that Re-Logic has to work on modder requests is 1.4.1.

**Comment by hakusaro** [comment]
(They understand the request and desire but have not had time to work on anything modders have sent)

**Comment by dark22** [comment]
No problem!  I was just looking for something about it.  To be honest, my expectation of seeing this working with SSC is great.  Thanks for the reply.

**Comment by hakusaro** [comment]
It's p possible -- I just don't know how much we're going to be able to get changed. I actually have pretty good confidence in this.

**Comment by dark22** [comment]
Taking a quick look at the game code after 1.4.1,  I realized there were no changes made to this? I'm afraid this may not happen, as there are a few additional updates on the way, but I hope I'm wrong!

**Comment by bartico6** [comment]
Yes, this was not changed in 1.4.1, sadly.

**Comment by hakusaro** [comment]
I have a bad feeling about this one.

**Comment by dark22** [comment]
> I have a bad feeling about this one.

Sadly, unfortunately it is something simple to implement with few changes to the network code. But they own the game and the decisions are theirs. However, I appreciate your attention and willingness to request a change to Re-Logic.

**Comment by Kieaer** [comment]
Here's 2024 years. still not working

**Comment by Finallf** [comment]
Is this option already working?

**Comment by ACaiCat** [comment]
> Is this option already working?

Yes, it works in TShock 6
